# enconding.test:
#    musl: FATAL ERROR: encoding 'koi8-ru' is not recognised: musl doesn't support non-utf8

diff -Naur cabextract-1.10/test/encoding.test cabextract-1.10-patched/test/encoding.test
diff --git a/test/encoding.test b/test/encoding.test
index 2ec3571..4e5ba12 100755
--- a/test/encoding.test
+++ b/test/encoding.test
@@ -7,7 +7,7 @@ if [ `"$cabextract" -h 2>&1 | grep -c encoding` -eq 0 ]; then
     exit 77
 fi
 
-"$cabextract" -e koi8-ru -l cabs/encoding-koi8.cab >$actual
+"$cabextract" -e koi8-r -l cabs/encoding-koi8.cab >$actual
 compare_with <<'EOF'
 Viewing cabinet: cabs/encoding-koi8.cab
  File size | Date       Time     | Name
